nvram.c

```diff
@@ -30,67 +30,67 @@
 /* Global variables */
 static int init = 0;
 static char temp[BUFFER_SIZE];
-static int firmae_nvram = 1;
+#define FIRMAE_NVRAM 1
 
-static int dir_lock() {
+static int _libinject_dir_lock() {
     int dirfd;
     // If not initialized, check for existing mount before triggering NVRAM init
     if (!init) {
         PRINT_MSG("%s\n", "Triggering NVRAM initialization!");
-        nvram_init();
+        libinject_nvram_init();
     }
 
     dirfd = open(MOUNT_POINT, O_DIRECTORY | O_RDONLY);
     if(dirfd < 0) {
        PRINT_MSG("Couldn't open %s\n", MOUNT_POINT);
     }
-    if(flock_asm(dirfd,LOCK_EX) < 0) {
+    if(_libinject_flock_asm(dirfd,LOCK_EX) < 0) {
        PRINT_MSG("Couldn't lock %s\n", MOUNT_POINT);
     }
     return dirfd;
 }
 
-static void dir_unlock(int dirfd) {
-    if(flock_asm(dirfd,LOCK_UN) < 0) {
+static void _libinject_dir_unlock(int dirfd) {
+    if(_libinject_flock_asm(dirfd,LOCK_UN) < 0) {
        PRINT_MSG("Couldn't unlock %s\n", MOUNT_POINT);
     }
     close(dirfd);
     return;
 }
 
-int true() {
+int libinject_ret_true() {
```

Comment:
I think we may have discussed this before, but maybe we want to change this to say "return 1" instead of "return true"? It can be confusing, because lots of C functions return 0 to indicate success and nonzero (-1) for errors.

Reply:
I like it, thanks